gwtlib/git_ops.py

```
@@ -3,6 +3,27 @@
 import sys
 
 
+def is_worktree_dirty(worktree_path: str, include_untracked: bool = True) -> bool:
```

Ruff (0.15.0)

[error] Boolean-typed positional argument in function definition

(FBT001)